########## Testcase 0: Failed ##########
Expected Output: 
Cycle doesn't exists.

Your Output: 
Exception in thread "main" java.util.NoSuchElementException
	at java.base/java.util.Scanner.throwFor(Unknown Source)
	at java.base/java.util.Scanner.next(Unknown Source)
	at java.base/java.util.Scanner.nextInt(Unknown Source)
	at java.base/java.util.Scanner.nextInt(Unknown Source)
	at DiG.<init>(Solution.java:59)
	at Solution.main(Solution.java:19)
----------------------------------------

########## Testcase 1: Failed ##########
Expected Output: 
Cycle doesn't exists.

Your Output: 
Exception in thread "main" java.util.NoSuchElementException
	at java.base/java.util.Scanner.throwFor(Unknown Source)
	at java.base/java.util.Scanner.next(Unknown Source)
	at java.base/java.util.Scanner.nextInt(Unknown Source)
	at java.base/java.util.Scanner.nextInt(Unknown Source)
	at DiG.<init>(Solution.java:59)
	at Solution.main(Solution.java:19)
----------------------------------------

########## Testcase 2: Failed ##########
Expected Output: 
Cycle exists.

Your Output: 
Exception in thread "main" java.util.NoSuchElementException
	at java.base/java.util.Scanner.throwFor(Unknown Source)
	at java.base/java.util.Scanner.next(Unknown Source)
	at java.base/java.util.Scanner.nextInt(Unknown Source)
	at java.base/java.util.Scanner.nextInt(Unknown Source)
	at DiG.<init>(Solution.java:59)
	at Solution.main(Solution.java:19)
----------------------------------------

########## Testcase 3: Failed ##########
Expected Output: 
Cycle exists.

Your Output: 
Exception in thread "main" java.util.NoSuchElementException
	at java.base/java.util.Scanner.throwFor(Unknown Source)
	at java.base/java.util.Scanner.next(Unknown Source)
	at java.base/java.util.Scanner.nextInt(Unknown Source)
	at java.base/java.util.Scanner.nextInt(Unknown Source)
	at DiG.<init>(Solution.java:59)
	at Solution.main(Solution.java:19)
----------------------------------------

########## Testcase 4: Failed ##########
Expected Output: 
Cycle exists.

Your Output: 
Exception in thread "main" java.util.NoSuchElementException
	at java.base/java.util.Scanner.throwFor(Unknown Source)
	at java.base/java.util.Scanner.next(Unknown Source)
	at java.base/java.util.Scanner.nextInt(Unknown Source)
	at java.base/java.util.Scanner.nextInt(Unknown Source)
	at DiG.<init>(Solution.java:59)
	at Solution.main(Solution.java:19)
----------------------------------------

########## Testcase 5: Failed ##########
Expected Output: 
Cycle exists.

Your Output: 
Exception in thread "main" java.util.NoSuchElementException
	at java.base/java.util.Scanner.throwFor(Unknown Source)
	at java.base/java.util.Scanner.next(Unknown Source)
	at java.base/java.util.Scanner.nextInt(Unknown Source)
	at java.base/java.util.Scanner.nextInt(Unknown Source)
	at DiG.<init>(Solution.java:59)
	at Solution.main(Solution.java:19)
----------------------------------------

########## Testcase 6: Failed ##########
Expected Output: 
Cycle doesn't exists.

Your Output: 
Exception in thread "main" java.util.NoSuchElementException
	at java.base/java.util.Scanner.throwFor(Unknown Source)
	at java.base/java.util.Scanner.next(Unknown Source)
	at java.base/java.util.Scanner.nextInt(Unknown Source)
	at java.base/java.util.Scanner.nextInt(Unknown Source)
	at DiG.<init>(Solution.java:59)
	at Solution.main(Solution.java:19)
----------------------------------------

########## Testcase 7: Failed ##########
Expected Output: 
Cycle exists.

Your Output: 
Exception in thread "main" java.util.NoSuchElementException
	at java.base/java.util.Scanner.throwFor(Unknown Source)
	at java.base/java.util.Scanner.next(Unknown Source)
	at java.base/java.util.Scanner.nextInt(Unknown Source)
	at java.base/java.util.Scanner.nextInt(Unknown Source)
	at DiG.<init>(Solution.java:59)
	at Solution.main(Solution.java:19)
----------------------------------------

